src/web/types/Portalapptype.ts

```diff
@@ -0,0 +1,14 @@
+import Abstractapptypes from "./Abstractapptypes";
+
+export interface searchParams {
+    orgUrl : string;
+    dataSource: string;
+    schema : string;
+    additionalParams: string | undefined;
+}
+
+export default interface portalapptypes extends Abstractapptypes {
+  entityName: string | undefined;
```

Comment:
These values are repeated across all app types. Shouldn't we move them to a common interface?